google-chart.html

```diff
@@ -338,19 +338,43 @@
     // Promise for the DataView with which the chart is drawn.
     _dataView: null,
 
+    // Promise for the drawn chart.
+    _chartDrawn: null,
+
+    // The current selection in the chart.
+    _selection: null,
+
     // The chart type was changed.
     // We need to create a new chart and redraw.
     _typeChanged: function() {
-      this._chart = this._createChart();
+      this._chart = createChart(this.type, this.$.chartdiv)
+          .then(this._setupChart.bind(this));
```

Comment:
Is this caveat considering the polyfill? This element has been using `iron-ajax` to load data which makes use of promises (via the polyfill). It may not be native but it should work.